### DISCUSSION TITLE
comprehensive medication/therapy data?

### DISCUSSION THREAD

**Discussion by ALINA991** [now-i-know-my-abcd/docs#81] [Where in the World is…? (Help)]
Hello all, 
I am wondering if any data concerning medication and therapy (start, duration, type, dosage, frequency) in between two recording session exists, specifically for ADHD (more comprehensive that the "last 2 weeks" as can be found in the medication inventory or "type of mental heath services... medication management" as in the ksads background)? Any help would be appreciated, thanks!

**Comment by clarefmccann** [comment]
Hi Alina! The only questions about services I can find are within the KSADS. These questions are still framed as "2 weeks," but there are more follow-up questions about the length: **ph_p_meds**. Hopefully, others can chime in with more information!

**Comment by emily-j-f** [comment]
Hi Alina, I'm just wondering if you were able to find any longer-term medication info? I'm also looking for ADHD meds info, but have only been able to find the data for medications in the 2 weeks prior to the assessment.
